libyul/optimiser/CommonSubexpressionEliminator.cpp

```diff
@@ -95,13 +95,13 @@ void CommonSubexpressionEliminator::visit(Expression& _e)
 
 	if (Identifier const* identifier = get_if<Identifier>(&_e))
 	{
-		YulString name = identifier->name;
-		if (m_value.count(name))
+		YulString identifierName = identifier->name;
```

Comment:
Why rename this?

Reply:
It was shadowing some variable, at least that was what my clang-tidy was warning about, so I renamed it.
I can rename back if you want

Reply:
Ah okay, no need to remove then.